### ISSUE TITLE
Improved accesibility for blind users

### ISSUE THREAD

**Issue by DeamonDavid** [phonegap/phonegap-plugin-barcodescanner#605]
### Expected Behaviour
When using the barcode scanner, screen readers such as VoiceOver (iOS) and TalkBack (Android) should be able to announce for blind users when a possible barcode is in focus, and guide the user to move the camera until it is in place.

On iOS, this kind of feature is available on the normal camera, when trying to take picture of a face.
Then it tells where the face is on the screen when moving the camera.

### Actual Behaviour
Nothing happens, except the piiping sound when the barcode is scanned.

#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
iOS 11.2.5

#### Cordova CLI version and cordova platform version

    cordova --version
    cordova platforms

#### Plugin version

    cordova plugin version | grep phonegap-plugin-barcodescanner

**Comment by DeamonDavid** [comment]
There is an app that hads this feature called Seeing AI. Maybe the concept used in that app can be used here...?
Check out the app and how it works on iOS app store here: https://itunes.apple.com/us/app/seeing-ai/id999062298?mt=8
